### ISSUE TITLE
govet: allow analyzers suite configuration

### ISSUE THREAD

**Issue by ernado** [golangci/golangci-lint#696]
Enabling or disabling go vet plug-ins would be very cool.
Seems like implementation is pretty straight-forward:
https://github.com/golangci/golangci-lint/blob/c9a92552385e742555fe341f813e6beba93defae/pkg/golinters/govet.go#L36-L61

We can filter analyzers in `NewGovet` function, but I'm not sure about config.
I think that there should be something like we have for `linters` section:
```yml
linters:
  enable:
    - megacheck
    - govet
  enable-all: false
  disable:
    - maligned
    - prealloc
  disable-all: false
```
But in `linters-settings.govet`.

Rel: #446
